scripts/MultiPlotting/multi_plotting_context.py

```diff
@@ -49,6 +50,12 @@ def add_annotate(self, subplotName, label):
     def add_vline(self, subplotName, xvalue, name):
         self.subplots[subplotName].add_vline(xvalue, name)
 
+    def removePlotLine(self, subplotName, name):
+        try:
+            self.subplots[subplotName].removePlotLine(name)
+        except:
```

Comment:
Would it be better to use a specific exception here, I think KeyError might do it?